src/bin/rcc.rs

```diff
@@ -102,16 +104,41 @@ fn real_main(
     } else {
         &opt.opt
     };
-    let (result, warnings) = compile(buf, &opt);
-    handle_warnings(warnings, file_id, file_db);
+    if !opt.jit {
+        let module = rcc::initialize_aot_module("rccmain".to_owned());
+        let (result, warnings) = compile(module, buf, &opt);
+        handle_warnings(warnings, file_id, file_db);
 
-    let product = result?;
-    if opt.no_link {
-        return assemble(product, output);
+        let product = result.map(|x| x.finish())?;
+        if opt.no_link {
+            return assemble(product, output);
+        }
+        let tmp_file = NamedTempFile::new()?;
+        assemble(product, tmp_file.as_ref())?;
+        link(tmp_file.as_ref(), output).map_err(io::Error::into)
+    } else {
+        let module = rcc::initialize_jit_module();
+        let (result, warnings) = compile(module, buf, &opt);
+        handle_warnings(warnings, file_id, file_db);
+        let mut rccjit = rcc::RccJIT::from_module(result?);
+        rccjit.finalize();
+        if let Some(main) = rccjit.get_compiled_function("main") {
+            let args = std::env::args();
```

Comment:
I think this will include `rcc` as the first argument, could you use `args()[1..]` instead?